### PR TITLE
[LANGUAGE UPDATE] Frontend translations for Italian and German

### PR DESCRIPTION
**Pull request by lukasitaly**
## What type of PR is this?

- cleanup
- translation

## What this PR does / why we need it:

This PR fully translates the frontend into German and Italian using GitHub instead of Weblate. Additionally, I have organized the strings and sections in the translation files alphabetically for improved readability and easier navigation. While this reordering is not strictly necessary for Weblate, it may help future contributors quickly locate and manage the strings.

## Which issue(s) this PR fixes:

None

## Testing

Not done yet

## Summary by CodeRabbit

- **New Features**
	- Enhanced German and Italian localization with updated translation keys for improved clarity and functionality.
	- Introduction of new user prompts and messages in the "global" and "index" sections to enhance user engagement and guidance.
	- Expanded "items" and "profile" sections with clearer instructions and detailed account management options.

- **Improvements**
	- Reorganized localization content for better readability and user experience.
	- Added meaningful translations for previously empty keys in the Italian localization.

### CODE REVIEW

File: frontend/locales/de.json [hunk 1]
```diff
@@ -74,54 +40,88 @@
...
-        "test": "Test",
         "gen_invite": "Einladungslink generieren",
-        "user_profile": "Benutzerprofil",
-        "user_profile_sub": "Benutzer einladen und Ihr Konto verwalten.",
-        "active": "Aktiv",
+        "group_settings": "Gruppeneinstellungen",
+        "group_settings_sub": "Geteilte Gruppeneinstellungen. Möglicherweise müssen Sie Ihren Browser aktualisieren, damit einige Einstellungen wirksam werden.",
         "inactive": "Inaktiv",
+        "new_password": "Neues Passwort",
         "notifier_modal": "{ type, select, true {Edit} false {Create} other {Other}} Melder",
```
Comment (tankerkiller125) [inline frontend/locales/de.json]: Edit, Create and Other should be updated with the language specific words to replace them.

File: frontend/locales/it.json [hunk 2]
```diff
@@ -74,54 +40,88 @@
...
+        "current_password": "Password Corrente",
+        "delete_account": "Elimina Account",
+        "delete_account_sub": "Elimina il tuo account e tutti i dati associati. Questa operazione non può essere annullata.",
+        "enabled": "Abilitato",
+        "gen_invite": "Genera Link di Invito",
+        "group_settings": "Impostazioni Gruppo",
+        "group_settings_sub": "Impostazioni Gruppo Condivise. Potrebbe essere necessario aggiornare il browser affinché alcune impostazioni vengano applicate.",
+        "inactive": "Inattivo",
+        "new_password": "Nuova Password",
+        "notifier_modal": "{ type, select, true {Edit} false {Create} other {Other}} Notifier",
```
Comment (tankerkiller125) [inline frontend/locales/it.json]: Same thing here, Edit, Create, and Other get replaced.

**Comment by lukasitaly** [comment]
Thank you or the review! I will replace them

**Comment by lukasitaly** [comment]
@tankerkiller125 done!

**Pull request merged**